tests/phpunit/includes/check-wp-functions-trait.php

```diff
@@ -43,6 +43,17 @@ protected function check_method( $md5, $version, ...$args ) {
 	 * @param string ...$args Function name or class and method name.
 	 */
 	protected function check_internal_method( $md5, ...$args ) {
-		$this->assertEquals( $md5, $this->md5( ...$args ), sprintf( 'The function %s() emulates a WordPress function, are you sure that it needs to be modified?', implode( '::', $args ) ) );
+		$this->assertSame( $md5, $this->md5( ...$args ), sprintf( 'The function %s() emulates a WordPress function, are you sure that it needs to be modified?', implode( '::', $args ) ) );
+	}
+
+	/**
+	 * Checks if a file has been modified.
+	 *
+	 * @param string $md5      Expected method md5.
+	 * @param string $filepath Filepath of the file to check.
+	 */
+	protected function check_file( $md5, $filepath ) {
```

Comment:
```suggestion
	protected function check_file( $md5, $version, $filepath ) {
```
I'm not against this PR. However it won't help for WooCommerce becuase it must be dedicated to WordPress files (as you need to check the version of your tested object to skip the test for old versions which you forgot to do here).